pipelines/metrics_pipeline_visit.yaml

```diff
@@ -1,4 +1,3 @@
 description: Visit metrics pipeline
 imports:
   - location: $FARO_DIR/pipelines/measurement/measurement_visit.yaml
-  - location: $FARO_DIR/pipelines/summary/summary_visit.yaml
```

Comment:
For some reason `metrics_pipeline_patch.yaml` still has a `summary_patch.yaml`, but none of the others retained their summary step.  I'm a little unclear what the goal is with the summary stuff.